en/components/grid.md

```diff
@@ -639,3 +638,17 @@ Our community is active and always welcoming to new ideas.
 
 * [Ignite UI for Angular **Forums**](https://www.infragistics.com/community/forums/f/ignite-ui-for-angular)
 * [Ignite UI for Angular **GitHub**](https://github.com/IgniteUI/igniteui-angular)
+
+## Grid Limitations
+
+<div ckass="devider"></div>
+
+### General
+
+|Limitation|Description|
+|--- |--- |--- |
+|Column widths into `percentage`|Currently we do not support mixing column widths with `%` and `px`.
+|Skipped `height` on the parent of the grid or grid itself| The `grid` is not rendered correctly whenever the `height` on the parent or on the grid itself is omitted.|
```

Comment:
When `height` of the grid's parent element or grid itself is not set | The `grid` height is not rendered correctly whenever the `height` of the grid's parent element or to the grid itself, is omitted.